tests/core20/compat/task.yaml

```diff
@@ -0,0 +1,11 @@
+summary: Ensure that core(16) compatibility is there
+
+execute: |
+    echo "Install test-snapd-sh (which uses the core snap)"
+    snap install test-snapd-sh
+
+    echo "Ensure that this pulled in core"
+    snap list | MATCH "^core +"
```

Comment:
could you just do `snap list core` instead?

Reply:
yes, thanks for the review